src/fides/api/ops/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -0,0 +1,71 @@
+from typing import List, Optional
+
+from fastapi import Depends, Security
+from fastapi_pagination import Page, Params
+from fastapi_pagination import paginate as fastapi_paginate
+from fastapi_pagination.bases import AbstractPage
+from loguru import logger
+from sqlalchemy.orm import Session
+from starlette.status import HTTP_200_OK
+
+from fides.api.ops.api import deps
+from fides.api.ops.api.v1 import scope_registry
+from fides.api.ops.api.v1 import urn_registry as urls
+from fides.api.ops.models.privacy_experience import ComponentType, PrivacyExperience
+from fides.api.ops.models.privacy_notice import PrivacyNotice, PrivacyNoticeRegion
+from fides.api.ops.schemas.privacy_experience import PrivacyExperienceResponse
+from fides.api.ops.util.api_router import APIRouter
+from fides.api.ops.util.oauth_util import verify_oauth_client
+
+router = APIRouter(tags=["Privacy Experience"], prefix=urls.V1_URL_PREFIX)
+
+
+@router.get(
+    urls.PRIVACY_EXPERIENCE,
+    status_code=HTTP_200_OK,
+    response_model=Page[PrivacyExperienceResponse],
+    dependencies=[
+        Security(verify_oauth_client, scopes=[scope_registry.PRIVACY_EXPERIENCE_READ])
+    ],
+)
+def privacy_experience_list(
+    *,
+    db: Session = Depends(deps.get_db),
+    params: Params = Depends(),
+    show_disabled: Optional[bool] = True,
+    region: Optional[PrivacyNoticeRegion] = None,
+    component: Optional[ComponentType] = None,
+    has_notices: Optional[bool] = None,
+) -> AbstractPage[PrivacyExperience]:
+    """
+    Return a paginated list of `PrivacyExperience` records in this system.
+    Includes some query params to help filter the list if needed
+    """
+    logger.info("Finding all Privacy Experiences with pagination params '{}'", params)
+    experience_query = db.query(PrivacyExperience)
+
+    if show_disabled is False:
+        experience_query = experience_query.filter(
+            PrivacyExperience.disabled.is_(False)
+        )
+    if region is not None:
+        experience_query = experience_query.filter(
+            PrivacyExperience.regions.contains([region])
+        )
+    if component is not None:
+        experience_query = experience_query.filter(
+            PrivacyExperience.component == component
+        )
+
+    results: List[PrivacyExperience] = []
+    for privacy_experience in experience_query.order_by(
+        PrivacyExperience.created_at.desc()
+    ):
+        privacy_notices: List[
```

Comment:
would we, for performance, want to skip this query if `has_notices` wasn't passed in?

Reply:
ah I had intentionally added it I thought it would be useful to have the notices embedded in the response as a default.

Reply:
that's a good point, sounds good 👍 

although the notices aren't embedded by default, right? you still have to pass in `has_notices` (since we check below for it, and its default value is `None`)?

Reply:
They are there by default! I get all of my up-front filtering down, and then I loop through every privacy experience and add the notices first.   And then I exclude the privacy experience + notices altogether if we only wanted experiences with notices and none existed.

Reply:
ohhh okay, I think I've been misinterpreting the name `has_notices` as more of `include_notices`. but `has_notices` makes sense, thanks for clarifying 👍